config/locales/en/shared.yml

```diff
@@ -75,6 +75,11 @@ en:
       apply_ref: 'Apply service case reference:'
       ccms_ref_html: "<abbr title='Client and Cost Management System'>CCMS</abbr> case reference:"
     assessment_results:
+      no_cfe_result:
+        heading: A caseworker needs to review your application
+        cannot_calculate: We cannot calculate your client's financial eligibility yet because you uploaded bank statements
```

Comment:
same here^^

Reply:
will make this change on the next ticket